src/mod/acme/acme.go

```diff
@@ -391,8 +430,18 @@ func (a *ACMEHandler) HandleRenewCertificate(w http.ResponseWriter, r *http.Requ
 		skipTLS = true
 	}
 
+	var dns bool
+
+	if dnsString, err := utils.PostPara(r, "dns"); err != nil {
```

Comment:
maybe something like  

```
dns := false

if dnsString, err := utils.PostPara(r, "dns"); err == nil && dnsString == "true" {
    dns = true
}
```

will be more easy to read?

Reply:
For your information, there is actually a function named `PostBool` in `mod/utils/utils.go` 😂

https://github.com/tobychui/zoraxy/blob/b4c771cdeeb59d63276416f99ecc9528bb5873af/src/mod/utils/utils.go#L63

Reply:
ah, didn't notice that func exist

we should use `PostBool` then, it even consider the upper-case